models/upload.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class Upload(Base):
```

Comment:
Here it is important to have methods to set status, check if finished, get upload and output path etc...